### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by wandergithub**
1. Added new column to db.
2. Inserted all the new data.
3. Updated species using transactions.
db-1
4. Deleted all records and ROLLBACK using transactions.
db-2
5. Deleted animals, used save points and rollbacks to update animal's weights. 
db-3
6. Finally, wrote all queries to answer questions:
db-4

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -2,8 +2,27 @@
...
+
+SELECT SUM(escape_attempts) FROM animals
+GROUP BY neutered;
+
+SELECT MAX(weight_kg), MIN(weight_kg) FROM animals
+GROUP BY species;
+
+SELECT AVG(escape_attempts) FROM animals
+WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01'
+GROUP BY species;
```
Comment (VICTOR-CHUKWU) [inline queries.sql:56]: - [x] Nice work @wandergithub , I can see screenshots of your transaction in your PR but I can't find the queries in your queries.sql, Kindly add it

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @wandergithub  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (wandergithub) [reply queries.sql:56]: Done 👍🏽

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Wander, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**